test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/ImageVersionIntegrationTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.cloud.integrationtests.smoke;
+
+import static org.assertj.core.api.SoftAssertions.assertSoftly;
+
+import java.time.Duration;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.kie.cloud.api.DeploymentScenarioBuilderFactory;
+import org.kie.cloud.api.deployment.CommandExecutionResult;
+import org.kie.cloud.api.deployment.Deployment;
+import org.kie.cloud.api.deployment.constants.DeploymentConstants;
+import org.kie.cloud.api.scenario.WorkbenchKieServerScenario;
+import org.kie.cloud.common.provider.KieServerClientProvider;
+import org.kie.cloud.integrationtests.AbstractCloudIntegrationTest;
+import org.kie.cloud.integrationtests.category.Smoke;
+import org.kie.cloud.integrationtests.util.TimeUtils;
+import org.kie.server.client.KieServicesClient;
+
+@Category(Smoke.class)
+public class ImageVersionIntegrationTest extends AbstractCloudIntegrationTest<WorkbenchKieServerScenario> {
+
+    private static final String KIE_VERSION = DeploymentConstants.getKieArtifactVersion();
+    private static final String KIE_API_ARTIFACT_NAME = "kie-api";
+    private static final String DEPLOYMENT_PATH = "/opt/eap/standalone/deployments";
+    private static final String ARTIFACT_VERSION_PATTERN_STRING = "(?:" + KIE_API_ARTIFACT_NAME + "-)(.*)\\.jar";
+    private static final Pattern ARTIFACT_VERSION_PATTERN = Pattern.compile(ARTIFACT_VERSION_PATTERN_STRING);
+
+    @Override
+    protected WorkbenchKieServerScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
+        return deploymentScenarioFactory.getWorkbenchKieServerScenarioBuilder().build();
+    }
+
+    @Test
+    public void testImageVersions() {
+        assertSoftly(softly -> {
+            softly.assertThat(getKieServerImageVersion()).isEqualTo(KIE_VERSION);
+            softly.assertThat(getImageVersion(deploymentScenario.getWorkbenchDeployment()))
+                  .as("Check that %s artifact is present with version %s", KIE_API_ARTIFACT_NAME, KIE_VERSION)
```

Comment:
I didn't know that as has this kind of "printf" functionality. Thanks for inspiration :)